module/applications/actor/character-sheet-2.mjs

```diff
@@ -439,12 +439,32 @@ export default class ActorSheet5eCharacter2 extends ActorSheet5eCharacter {
       if ( data.custom ) data.custom.split(";").forEach(v => values.push({ label: v.trim() }));
       if ( values.length ) traits[trait] = values;
     }
+
     // Combine damage & condition immunities in play mode.
     if ( (this._mode === this.constructor.MODES.PLAY) && traits.ci ) {
       traits.di ??= [];
       traits.di.push(...traits.ci);
       delete traits.ci;
     }
+
+    // Prepare damage modifications
+    const dm = this.actor.system.traits?.dm;
+    if ( dm ) {
+      const rollData = this.actor.getRollData({ deterministic: true });
+      const values = Object.entries(dm.amount).map(([k, v]) => {
+        const total = simplifyBonus(v, rollData);
+        if ( !total ) return null;
+        const value = {
+          label: `${CONFIG.DND5E.damageTypes[k]?.label ?? key} ${formatNumber(total, { signDisplay: "always" })}`,
+          color: total < 0 ? "maroon" : "green"
```

Comment:
Should this be the other way around? Reducing damage taken is a good thing, right?